mnist/README.md

```diff
@@ -216,51 +138,55 @@ argo submit model-train.yaml -n ${NAMESPACE} --serviceaccount tf-user \
 Your training workflow should now be executing.
 
 You can verify and keep track of your workflow using the argo commands:
+
 ```
-$ argo list
+$ argo -n ${NAMESPACE} list
 NAME                STATUS    AGE   DURATION
 tf-workflow-h7hwh   Running   1h    1h
 
-$ argo get tf-workflow-h7hwh
+$ argo -n ${NAMESPACE} get tf-workflow-h7hwh
 ```
 
-## Monitoring
+After the STATUS to `Succeeded`, then you can use it.
 
-There are various ways to monitor workflow/training job. In addition to using `kubectl` to query for the status of `pods`, some basic dashboards are also available.
 
-### Argo UI
+## 9.Submit serving workflow[optional]
```

Comment:
What is the serving workflow? And why does model serving need to be disabled?

Reply:
this example already has this feature and I think it is reasonable to separate the train and serving model. I did not change it so there is not diff.

Reply:
Ok thanks.